djstripe/stripe_objects.py

```diff
@@ -954,6 +954,59 @@ def api_retrieve(self, api_key=None):
         return stripe_event
 
 
+class StripePayout(StripeObject):
+    class Meta:
+        abstract = True
+
+    stripe_class = stripe.Payout
+    stripe_dashboard_item_name = "payouts"
+
+    amount = StripeCurrencyField(
+        help_text="Amount to be transferred to your bank account or debit card."
+    )
+    arrival_date = StripeDateTimeField(
+        help_text=(
+            "Date the payout is expected to arrive in the bank. "
+            "This factors in delays like weekends or bank holidays."
+        )
+    )
+    currency = StripeCharField(max_length=3, help_text="Three-letter ISO currency code.")
+    failure_code = StripeCharField(
+        max_length=23,
+        blank=True, null=True,
+        choices=enums.PayoutFailureCode.choices,
+        help_text="Error code explaining reason for transfer failure if available. "
+        "See https://stripe.com/docs/api/python#transfer_failures."
+    )
+    failure_message = StripeTextField(
+        null=True, blank=True,
+        help_text="Message to user further explaining reason for payout failure if available."
+    )
+    method = StripeCharField(
+        max_length=8,
```

Comment:
Could we derive the maximum length from the enum, so that we don't get tripped up here in the future?  Makes sense to have a helper method (or calculate it and store it in enum) so we can use it elsewhere.